### ISSUE TITLE
Customization of loaded contribs

### ISSUE THREAD

**Issue by quicklisp** [quicklisp/quicklisp-slime-helper#10]
Right now slime-setup is called with a fixed list of contribs `(slime-fancy)`. It would be nice to have that list configurable.

**Comment by alessiostalla** [comment]
Yes! Do you have a design in mind? I can do the actual coding if you want.
